src/libasr/pass/print_arr.cpp

```diff
@@ -122,7 +122,7 @@ class PrintArrVisitor : public PassUtils::PassVisitor<PrintArrVisitor>
             }
             print_stmt = ASRUtils::STMT(ASR::make_Print_t(
                 al, x.base.base.loc, nullptr, body.p, body.size(),
-                nullptr, nullptr));
+                x.m_separator, x.m_end));
```

Comment:
Could you add consideration for `sep` and `end `as keyword arguments.
Check that `lpython` does not give an error when order is not followed -
```
def main():
    print(sep = "++","c", 3, "d", 4, [4, 5, 6])

main()

(lp) C:\Users\kunni\lpython>python try.py
  File "C:\Users\kunni\lpython\try.py", line 2
    print(sep = "++","c", 3, "d", 4, [4, 5, 6])
                                              ^
SyntaxError: positional argument follows keyword argument

(lp) C:\Users\kunni\lpython>src\bin\lpython try.py
c++3++d++4++[4, 5, 6] 
```

Reply:
Could be corrected in another pull request if this is close to merging. I would always prefer that the issue is fixed completely. Thanks !

Reply:
But this should have been caught in AST. I'm surprised that we missed handling this.
Thanks, @faze-geek for reporting this. Can you please report this as a new Parser issue?

Reply:
Sure I'll do the needful quick !